metadata.json

```diff
@@ -18,7 +18,7 @@
     },
     {
       "name": "puppetlabs-apt",
-      "version_requirement": ">= 7.0.1 < 9.0.0"
+      "version_requirement": ">= 7.0.1 < 10.0.0"
```

Comment:
From what I can tell the apt 9.0 release introduced a breaking change to the `apt::ppa` class: https://github.com/puppetlabs/puppetlabs-apt/commit/83082c088e6a6703a08071fbd5af0297beae62b9?w=1. However, this module doesn't use that class so this bump should be ok. That said, it would be good to test that a 5.5.x agent can upgrade to 7.x with the updated apt module.

Reply:
I'll give that a whirl in the lab tomorrow

Reply:
@joshcooper slight fly  in the ointment, i cant produce a puppet compatible CA in my puppet 7 labs in order to test this senario